puffin/src/fuzzer/stats_stage.rs

```diff
@@ -238,6 +238,7 @@ where
     EM: UsesState<State = Z::State>,
     Z: Evaluator<E, EM>,
 {
+    #[allow(dead_code)]
     pub fn new() -> Self {
```

Comment:
The [rust API guidelines](https://rust-lang.github.io/api-guidelines/interoperability.html?highlight=default#types-eagerly-implement-common-traits-c-common-traits) suggest having both `Default` and `new()` in that case, but I'm unsure how useful this advise is in our case because this particular struct is private to our crate.

I think for now I would like to keep `new()` and also implement `Default` where those occur, as you suggested.

On this particular instance, do you remember why the addition of `StatsStage` in the fuzzer pipeline is commented out? Should we keep it at all?

Reply:
I added implementations of `Default` where we had a parameter-less `fn new() -> Self`. See 722015919adc95ffd79477e1fbb010efd6699388 for the changes.